ces-poc-demo.test/WeatherForecastTests.cs

```diff
@@ -0,0 +1,88 @@
+using ces_poc_demo.Controllers;
+using ces_poc_demo.Implementation;
+using ces_poc_demo.Tests;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace ces_poc_demo.test
+{
+    public class WeatherForecastTests
+    {
+        WeatherForecastController oWeather;
+
+        public WeatherForecastTests() 
+        {
+            List<WeatherForecast> forecasts = [
+                new WeatherForecast() { ID = 1, Summary = "Freezing", TemperatureC = 4 },
+                new WeatherForecast() { ID = 2, Summary = "Chilly", TemperatureC = 10 },
```

Comment:
Try to avoid static data for tests, you can you auto fixture/bogus

Reply:
added data using autofixtures